src/pages/Delegate/index.tsx

```diff
@@ -113,11 +114,28 @@ export const Delegate = () => {
       ;(await tx)
         .signSubmitAndWatch(selectedAccount?.polkadotSigner)
         .subscribe((event) => {
-          console.info(event)
+          let msg: string
+          switch (event.type) {
+            case 'finalized':
+              msg = `Tx - Finalized. Block Number: ${event.block.number}. TxHash: ${event.txHash}`
```

Comment:
This is much better IMHO beside the tx hash
```suggestion
              msg = `Tx finalized in block ${event.block.number}`
```